cmake/templates/cmake_variables.rst.in

```diff
@@ -10,6 +10,61 @@
    This file has been automatically generated using the CMake tool.
    Do not edit manually.
 
+.. _important_cmake_options:
```

Comment:
Would it not be good to put this list somewhere more prominent, like just before this section: https://hpx-docs.stellar-group.org/branches/master/html/manual/building_hpx.html#platform-specific-build-recipes? You could then also include a link from this section to the full list: https://hpx-docs.stellar-group.org/branches/master/html/manual/building_hpx.html#cmake-variables-used-to-configure-hpx.